src/vmm/src/cpuid/common.rs

```diff
@@ -23,7 +22,7 @@ pub enum GetCpuidError {
 ///
 /// - When the given `leaf` is more than `max_leaf` supported by CPUID.
 /// - When the the CPUID leaf `sub-leaf` is invalid (all its register equal 0).
-#[cfg(cpuid)]
+
```

Comment:
are blank lines like this meaningful?

Reply:
No, they are leftover from find/replace I will remove them.